homeassistant/util/__init__.py

```diff
@@ -218,13 +218,13 @@ def __call__(self, method: Callable) -> Callable:
 
             async def throttled_value() -> None:
                 """Stand-in function for when real func is being throttled."""
-                return None
+                return None  # pylint: disable=hass-return-none
```

Comment:
Why disable it here, you should be able to remove the None?

Reply:
I think this is a very special case when function was created to return only `None` value. Although it can be removed the idea of that function that it returns `None` instead of no return value.